starwars/src/Components/CharCard/CharCard.css

```diff
@@ -0,0 +1,11 @@
+.topCard{
```

Comment:
Make sure you're adding spaces between the selector and `{` and `display:` `flex`